types/jasmine/jasmine-tests.ts

```diff
@@ -179,6 +179,11 @@ describe("Included matchers:", () => {
         expect(foo).toThrowError(TypeError, "foo bar baz");
     });
 
+    it("The 'toBeInstanceOf' matcher is for testing an object is an instance of the supplied class/constructor", () => {
+        class Foo {}
+        expect(new Foo()).toBeInstanceOf(Foo);
```

Comment:
Please add a few cases:
- for `Number` and `Boolean` built-in types
- negative case when you use not a prototype and expect compiler warning on that line